docker-compose.yml

```diff
@@ -4,11 +4,16 @@ services:
       context: .
       dockerfile: Dockerfile
       target: development
+    volumes:
+      - .:/usr/src/app
+      - /usr/src/app/node_modules
```

Comment:
This configuration:
   1. Mounts your local project directory to /usr/src/app
   2. Creates an anonymous volume for /usr/src/app/node_modules, which isolates the container's node_modules from your local one